File: Climbing Stairs/Climbing Stairs.py

def climbStairs(self, n):
    """
    :type n: int
    :rtype: int
    """
    cur = 1
    pre = 1
    temp = 0
    
    for i in range(1, n):
        temp = cur
        cur += pre
        pre = temp
        
    return cur

File: Climbing Stairs/Climbing Stairs.php
<?php

function climbStairs($n) {
    $cur = 1;
    $pre = 1;
    $temp = 0;

    for ($i = 1; $i<$n; $i++){
        $temp = $cur;
        $cur += $pre;
        $pre = $temp;
    }
        
    return $cur;
}

print(climbStairs(3));

?>